src/resources/views/layouts/app.blade.php

```diff
@@ -57,38 +57,34 @@
                 <div class="collapse navbar-collapse" id="navbarSupportedContent">
                     <!-- Left Side Of Navbar -->
                     <ul class="navbar-nav mr-auto">
-
                     </ul>
-
                     <!-- Right Side Of Navbar -->
                     <ul class="navbar-nav ml-auto">
-                        <!-- Authentication Links -->
-                        <!-- TODO -->
-                        <!-- Remove ability to self-register once admin register is implemented -->
                         @guest
-                            @if (Route::has('register'))
-                                <li class="nav-item">
-                                    <a class="nav-link" href="{{ route('register') }}">{{ __('Register') }}</a>
-                                </li>
-                            @endif
-                            <!-- Needs to be something here to only show admins option to create quizzes and users -->
+                            <!-- Guests see nothing here -->
                         @else
                             <li class="nav-item dropdown">
                                 <a id="navbarDropdown" class="nav-link dropdown-toggle" href="#" role="button"
                                     data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" v-pre>
                                     {{ Auth::user()->first_name }} {{ Auth::user()->last_name }}
                                 </a>
-                                <!-- TODO -->
-                                <!-- navbar items once logged in - differentiate between student/admin -->
                                 <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
-                                    <!-- admin only -->
-                                    <a class="dropdown-item" href="{{ url('add_user') }}">
-                                        {{ __('Add Users') }}
-                                    </a>
-                                    <a class="dropdown-item" href="{{ url('create_quiz') }}">
-                                        {{ __('Create New Quiz') }}
-                                    </a>
-                                    <!--  -->
+                                    <!-- Admin and TA -->
+                                    @if (Bouncer::is(Auth::user())->an("admin") || Bouncer::is(Auth::user())->a("ta"))
```

Comment:
I just discovered, this can be further simplified to
`@if (Bouncer::is(Auth::user())->a('admin', 'ta'))`